packages/react-reconciler/src/ReactFiberLane.old.js

```diff
@@ -635,15 +634,6 @@ export function markRootFinished(root: FiberRoot, remainingLanes: Lanes) {
 
   root.entangledLanes &= remainingLanes;
 
-  if (enableCache) {
-    const pooledCacheLanes = (root.pooledCacheLanes &= remainingLanes);
-    if (pooledCacheLanes === NoLanes) {
```

Comment:
I think this check got accidentally removed. The reason it's important is that we don't want to release the pooledCache until the shell commits, which allows ownership to transfer either the Suspense boundary or to the mounted Cache boundary.

I would expect this to cause test failures (not sure why CI didn't run tests for this PR, huh) but maybe we're missing a test or maybe there's some other change I'm not accounting for.

Reply:
Good catch, thanks for flagging. I think things are working anyway because ownership never really _moves_ from the root to a Suspense or Cache boundary (not without getting captures somewhere like `getSuspendedCachePool()`). There are definitely missing tests around various combinations of suspense / transition sequences, i'm working on adding those so that should give us more confidence. Once those tests are ready i'll confirm whether removing this again breaks any tests.